src/main/java/io/cloudsoft/spark/SparkNodeSshDriver.java

```diff
@@ -175,7 +175,7 @@ public void launch() {
                     + ">> /etc/hosts'").machine(brooklyn.location.basic.Locations.findUniqueSshMachineLocation(masterNode.getLocations()).get()));
                 // and on worker to know of master
                 brooklyn.util.task.DynamicTasks.queue(brooklyn.entity.software.SshEffectorTasks.ssh("sudo sh -c '"
-                    + "echo "+masterNode.getAttribute(Attributes.SUBNET_ADDRESS)+" "+masterNode.getAttribute(SparkNode.MASTER_FULL_HOSTNAME)+" "+masterNode.getAttribute(SparkNode.MASTER_SHORT_HOSTNAME)+" "
+                    + "echo "+masterNode.getAttribute(Attributes.HOSTNAME)+" "+masterNode.getAttribute(SparkNode.MASTER_FULL_HOSTNAME)+" "+masterNode.getAttribute(SparkNode.MASTER_SHORT_HOSTNAME)+" "
```

Comment:
if `HOSTNAME` is not an IP address i think this will fail, won't it?  `/etc/hosts` wants `IP hostname1 hostname2` format.

replace with `ADDRESS` if we want the public address rather than the internal?